extra-doc/docbook-sgml/spec

```diff
@@ -1,6 +1,5 @@
 VER=4.5
```

Comment:
Reset version to 0 to represent transitionalisation. Same applies for all below.